src/pages/standup.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { FC } from 'react';
+import Head from '@/components/head';
+import Layout from '@/components/Layout';
+
+import styles from '@/styles/standup.module.scss';
+
+const StandUp: FC = () => {
+    return (
+        <>
+            <Layout>
+                <Head title="Standup" />
+                <div className="container">
+                    <div className={styles.standupContainer}>
+                        <div className={styles.standupBanner}>
+                            <p>
+                                You have
+                                <span data-testid="missed-updates">
+                                    2 missed
+                                </span>
+                                Standup updates this week
+                            </p>
+                            <p>Let&apos;s try to avoid having zero days </p>
+                            <div className={styles.buttonContainer}>
+                                <button className={styles.continueButton}>
+                                    continue
+                                </button>
+                                <button className={styles.updateButton}>
+                                    Fill Old Updates
+                                </button>
+                            </div>
+                        </div>
+                        <div className={styles.standupUpdateContainer}>
```

Comment:
what is this exact is it a form or just few input fields?

Reply:
fixed